libs/sdk-core/src/persist/sync.rs

```diff
@@ -156,7 +156,8 @@ impl SqliteStorage {
           bitcoin_address,
           refund_tx_id
          FROM remote_sync.swap_refunds
-         WHERE bitcoin_address NOT IN (SELECT bitcoin_address FROM sync.swap_refunds);",
+         WHERE bitcoin_address NOT IN (SELECT bitcoin_address FROM sync.swap_refunds)
+         ON CONFLICT DO NOTHING;",
```

Comment:
Actually, I have one comment: Isn't the `WHERE ... NOT IN` clause redundant now?

Reply:
I agree. Removed that line in c80cb9f8dd3a19bd2a0114cbb6a8e9cbea8b616f